x-pack/plugins/lens/public/metric_visualization/metric_config_panel/title_align_option.tsx

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import { EuiFormRow, EuiSuperSelect } from '@elastic/eui';
+import { MetricState } from '../../../common/expressions';
+
+export interface TitlePositionProps {
+  state: MetricState;
+  setState: (newState: MetricState) => void;
+}
+
+const titleAlignPositions = [
+  { id: 'top', label: 'Top', direction: 'column' },
+  { id: 'bottom', label: 'Bottom', direction: 'column' },
+  { id: 'middle', label: 'Middle', direction: 'column' },
+  { id: 'left', label: 'Left', direction: 'row' },
+  { id: 'right', label: 'Right', direction: 'row' },
+  { id: 'center', label: 'Center', direction: 'row' },
+];
+
+export const TitleAlignOptions: React.FC<TitlePositionProps> = ({ state, setState }) => {
+  const direction = ['top', 'bottom'].includes(state.titlePosition ?? 'top') ? 'column' : 'row';
+
+  return (
+    <EuiFormRow
+      display="columnCompressed"
+      label={
+        <>
+          {i18n.translate('xpack.lens.metricChart.titleAlignLabel', {
+            defaultMessage: 'Title align',
```

Comment:
Per previous comments, change this label if it will affect both metric title and value alignment.